### ISSUE TITLE
Stucking at analysing model performance

### ISSUE THREAD

**Issue by bgyuris** [AI-sandbox/gnomix#46]
Hi, 

When i run gnomix it stuck at this stage. Seemingly no cpu usage...

Launching in training mode...
Reading vcf file...
Getting genetic map info...
Getting sample map info...
Building founders...
Splitting sample map...
Running Simulation...
Training...
Reading data...
Building model...
Training base models...
100%|████████████████████████████████████████| 373/373 [00:02<00:00, 131.65it/s]                                                                                                                     
Training smoother...
/home/install/anaconda3/lib/python3.9/site-packages/torch/autograd/__init__.py:251: UserWarning: CUDA initialization: The NVIDIA driver on your system is too old (found version 11040). Please update your GPU driver by downloading and installing a new version from the URL: http://www.nvidia.com/Download/index.aspx Alternatively, go to: https://pytorch.org to install a PyTorch version that has been compiled with your version of the CUDA driver. (Triggered internally at ../c10/cuda/CUDAFunctions.cpp:108.)
  Variable._execution_engine.run_backward(  # Calls into the C++ engine to run the backward pass
Evaluating model...
Re-training base models...
100%|████████████████████████████████████████| 373/373 [00:03<00:00, 123.36it/s]                                                                                                                     
Analyzing model performance...

**Comment by bgyuris** [comment]
Without modifying any parameters in config.yaml:

--------------------------------------------------------------------------------
-----------------------------------  Gnomix  -----------------------------------
--------------------------------------------------------------------------------
When using this software, please cite: 
Helgi Hilmarsson, Arvind S Kumar, Richa Rastogi, Carlos D Bustamante, 
Daniel Mas Montserrat, Alexander G Ioannidis: 
"High Resolution Ancestry Deconvolution for Next Generation Genomic Data" 
https://www.biorxiv.org/content/10.1101/2021.09.19.460980v1
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
Launching in training mode...
Reading vcf file...
Getting genetic map info...
Getting sample map info...
Building founders...
Splitting sample map...
Running Simulation...
Training...
Reading data...
Building model...
Training base models...
100%|████████████████████████████████████████| 373/373 [00:03<00:00, 113.20it/s]                                                                                                                            
Training smoother...
[13:05:10] WARNING: /workspace/src/learner.cc:480: 
Parameters: { use_label_encoder } might not be used.

  This may not be accurate due to some parameters are only used in language bindings but
  passed down to XGBoost core.  Or some parameters are not used but slip through this
  verification. Please open an issue if you find above cases.


Evaluating model...
Re-training base models...
100%|████████████████████████████████████████| 373/373 [00:03<00:00, 99.47it/s]                                                                                                                             
Analyzing model performance...

**Comment by lucagamam** [comment]
I have exactly the same issue
